packages/dhis2/test/index.test.js

```diff
@@ -149,22 +149,43 @@ describe('get', () => {
   });
 });
 
-describe('helperfunctions', () => {
+describe.only('helperfunctions', () => {
   it('should use the new tracker for enrollments', () => {
     const result = shouldUseNewTracker('enrollments');
     expect(result).to.be.true;
   });
 
   it('should use the new tracker for events', () => {
+    const result = shouldUseNewTracker('events');
+    expect(result).to.be.true;
+  });
+
+  it('should use the new tracker for relationships', () => {
+    const result = shouldUseNewTracker('relationships');
+    expect(result).to.be.true;
+  });
+
+  it('should use the new tracker for new tracker events', () => {
     const result = shouldUseNewTracker('tracker/events');
     expect(result).to.be.true;
   });
 
+  it('should use the new tracker for trackedEntityInstance', () => {
```

Comment:
This test fails right now, but I think it should pass.

I don't think we have support right now to map `trackedEntityInstances` to `tracker/trackeEntities`. But I think we should?

We support `create('enrollments')` and `create ('trackedEntities')`, so why not `create ('trackedEntitiesInstances)`?